.test/test.sh

```diff
@@ -12,5 +12,27 @@ set -- docker:cli docker:dind docker:windowsservercore notary # a little bit of
 # (see "library/" and ".external-pins/" for where these come from / are hard-coded for consistent testing purposes)
 # NOTE: we are explicitly *not* pinning "golang:1.19-alpine3.16" so that this also tests unpinned parent behavior (that image is deprecated so should stay unchanging)
 
+time bashbrew fetch "$@"
```

Comment:
Splitting this out from being implied by `sources.sh` would've made PRs like #16 more interesting (because we'd see the actual change in generation time, not affected by Git fetch time), but we're doing such a small set here that it's really not much of a difference. :smile: